veggieseasons/lib/screens/home.dart

```diff
@@ -3,52 +3,62 @@
 // found in the LICENSE file.
 
 import 'package:flutter/cupertino.dart';
-import 'package:veggieseasons/screens/favorites.dart';
-import 'package:veggieseasons/screens/list.dart';
-import 'package:veggieseasons/screens/search.dart';
-import 'package:veggieseasons/screens/settings.dart';
+import 'package:go_router/go_router.dart';
 
 class HomeScreen extends StatelessWidget {
-  const HomeScreen({super.key, this.restorationId});
+  const HomeScreen({
+    super.key,
+    this.restorationId,
+    required this.child,
+    required this.onTap,
+  });
 
   final String? restorationId;
+  final Widget child;
+  final void Function(int) onTap;
 
   @override
   Widget build(BuildContext context) {
+    final index = _getSelectedIndex(GoRouter.of(context).location);
     return RestorationScope(
       restorationId: restorationId,
-      child: CupertinoTabScaffold(
```

Comment:
`CupertinoTabScaffold` is gone, and is replaced by a `CupertinoPageScaffold` that contains the `ShellRoute` child and the `CupertinoTabBar` instead.